index.html

```diff
@@ -0,0 +1,86 @@
+<html>
+
+    <head>
+        <title>hannah kern</title>
+    </head>
+
+    <h1>
+        hannah kern
+    </h1>
+
+    <h1>
+        about
+    </h1>
+
+    <h1>
+        projects
+    </h1>
+
+    <h1>
+        contact
+    </h1>
+
+    <h1>
+        hello.
+    </h1>
+
+    <h2>
+        my name is hannah. i am passionate about helping people through innovative design. 
+    </h2>
+
+    <img src = "https://photos.google.com/photo/AF1QipNyZ9mTBYE373cA3-qUuMqOGml62suxp6QN_NCJ">
+
+    <img src = "https://photos.google.com/photo/AF1QipNYwca5fK7wrKjU4tJbM6ZT5NnxRMlUzUSdTZFV">
```

Comment:
These images don't work for me... Also, you should include them in your repo and use a relative path. I would not depend on google for these.